core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
Alphabetize. Please read the [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css) if you haven't already, and ensure that your code conforms to it.